perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -96,6 +96,26 @@ sub check_payment_redirect_id : Private {
     $c->stash->{report} = $p;
 }
 
+sub get_pending_subscription : Private {
+    my ($self, $c) = @_;
+
+    my $uprn = $c->stash->{property}{uprn};
+    my $len = length $uprn;
+    my $subs = $c->model('DB::Problem')->search({
```

Comment:
Restrict to body?

Reply:
This also feels a bit fragile, but not sure I have anything to suggest.

Reply:
Yes, it's not ideal.